samples/captioning/multi-track-captions.html

```diff
@@ -16,20 +16,23 @@
         function startVideo() {
             var url = FRAGMENTED_CAPTION_URL,
                 context,
-                player;
+                player,
+                TTMLRenderingDiv;
 
             videoElement = document.querySelector(".dash-video-player video");
+            TTMLRenderingDiv = document.querySelector("#ttml-rendering-div");
             context = new Dash.di.DashContext();
             player = new MediaPlayer(context);
             player.startup();
             player.attachView(videoElement);
+            player.attachTTMLRenderingDiv(TTMLRenderingDiv);
             player.setAutoPlay(true);
             player.attachSource(url);
             player.addEventListener(MediaPlayer.events.TEXT_TRACK_ADDED, onTrackAdded);
         }
 
         function onTrackAdded(e){
-            var selectMenu = document.getElementById("caption-menu");
+            var selectMenu = document.getElementById("caption-menu"),
```

Comment:
Selecting "Turn Caption OFF" throws "Uncaught TypeError: Cannot read property 'start' of undefined" at dash.debug.js:13742 which is virtualbuffer.js:findChunksForRange.